portfolio/src/main/webapp/script.js

```diff
@@ -186,6 +191,41 @@ function checkSubmit(inputElementId, submitElementId) {
 }
```

Comment:
the walkthrough mentions preventing users who haven't logged in from leaving a comment. Definitely not required that you do that in this CL, but is that something you're planning to do in the future?

Reply:
Disregard. Just saw pr #20